tools/ansible/tasks/wsu/main.yaml

```diff
@@ -20,25 +20,151 @@
         msg: "Cluster not patched for hybrid overlay"
       when: "hybrid_query.stdout == ''"
 
+    - name: Get kubernetes version
+      shell: "oc version -o json | jq -r '.serverVersion.gitVersion'"
```

Comment:
Good point, also make sure that you're getting the entire version and store it a variable so that you can avoid making unnecessary api calls.

Reply:
It was already registered in a variable, added failed_when condition
There was another API call not added in my PR, in task "Get kubernetes minor version" - I have changed it so it now parses the input from the variable instead of making another API call